packages/contracts/test/DepositFeed.ts

```diff
@@ -0,0 +1,155 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Contract, Signer, BigNumber } from 'ethers'
+import { applyL1ToL2Alias } from '@eth-optimism/core-utils'
+
```

Comment:
What happened to the idea of using Foundry for testing?

Reply:
it doesn't easily support testing events, which is 90% of what we need to test.